assets/js/main.js

```diff
@@ -1,12 +1,47 @@
-$(document).ready(function(){
-    $('.carousel').carousel();
-  });
-
- $('.dropdown-trigger').dropdown({
-    coverTrigger: false,
-    constrainWidth: false
-  });
-
-  $(document).ready(function(){
-    $('.sidenav').sidenav();
-  });
\ No newline at end of file
+$(document).ready(function () {
+  $('.carousel').carousel();
+});
+
+$('.dropdown-trigger').dropdown({
+  coverTrigger: false,
+  constrainWidth: false
+});
+
+$(document).ready(function () {
+  $('.sidenav').sidenav();
+});
+
+
+
+
+var loadFavorite = function () {
+  var savedFavorites = localStorage.getItem("favoriteArt");
+  if (!savedFavorites) {
+    return [];
+  }
+  savedFavorites = JSON.parse(savedFavorites);
+  return savedFavorites;
+}
+
+
+var saveFavorite = function (artwork) {
+  var savedFavorites = loadFavorite();
+  savedFavorites.push(artwork);
+  localStorage.setItem("favoriteArt", JSON.stringify(savedFavorites));
+
+}
+
```

Comment:
const someArt and below is test code to make sure your functions are working. Test code should be removed before pushing